libraries/ms-common/src/stm32f0xx/gpio_it.c

```diff
@@ -34,6 +35,14 @@ static uint8_t prv_get_irq_channel(uint8_t pin) {
   return 7;
 }
 
+StatusCode gpio_it_get_edge(const GpioAddress *address, InterruptEdge *edge) {
+  if (s_gpio_it_interrupts[address->pin].callback != NULL) {
```

Comment:
This is a check for most other functions in gpio_it.c, and I'm inclined to say it's not particularly non-obvious? adding bool registered would mean changing two other checks and also adding some cleanup code in case of bad interrupt register, so I think this is fine but if you still want it I'll add it